parachains/common/src/impls.rs

```diff
@@ -71,17 +71,18 @@ where
 
 /// A `HandleCredit` implementation that naively transfers the fees to the block author.
 /// Will drop and burn the assets in case the transfer fails.
-pub struct AssetsToBlockAuthor<R>(PhantomData<R>);
-impl<R> HandleCredit<AccountIdOf<R>, pallet_assets::Pallet<R>> for AssetsToBlockAuthor<R>
+pub struct AssetsToBlockAuthor<R, I = ()>(PhantomData<(R, I)>);
```

Comment:
```suggestion
pub struct AssetsToBlockAuthor<R, I>(PhantomData<(R, I)>);
```
I am not sure about the default here. Could accidentally reference the wrong instance AFAIK.